config/routes.rb

```diff
@@ -3,12 +3,18 @@
   root 'sessions#new'
 
   get 'login', to: 'sessions#new'
-  post 'login', to: 'sessions#create'
+  post 'login', to: 'sessions#create', as: 'post_login'
   get 'logout', to: 'sessions#destroy'
 
   get 'signup', to: 'users#new'
   post 'signup', to: 'users#create'
 
+  resources :users do
+    member do
+      get :confirm_email
+    end
+  end
+
```

Comment:
Okay, so this is how the actual verification goes through. 
That `member do` is what creates a new controller action in the `users` controller, which is is called "confirm_email". 
When they click on the link in the email, it actually leads them towards this particular URL, which is responsible for actually confirming the email.